worker/buildbot_worker/base.py

```diff
@@ -320,6 +320,16 @@ def remote_getWorkerInfo(self):
                 if os.path.isfile(filename):
                     with open(filename, "r") as fin:
                         files[f] = fin.read()
+
+        if os.path.exists('/etc/os-release'):
+            with open('/etc/os-release', "r") as fin:
+                for line in fin.readlines():
```

Comment:
```suggestion
                for line in fin:
```
readlines() return a list, and thus will load the whole file before the loop, while the suggested way, we load during the loop